src/Sentry.Unity/Integrations/UnityApplicationLoggingIntegration.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using Sentry.Integrations;
+using UnityEngine;
+
+namespace Sentry.Unity.Integrations
+{
+    internal sealed class UnityApplicationLoggingIntegration : ISdkIntegration
+    {
+        internal readonly ErrorTimeDebounce ErrorTimeDebounce = new(TimeSpan.FromSeconds(1));
+        internal readonly LogTimeDebounce LogTimeDebounce = new(TimeSpan.FromSeconds(1));
+        internal readonly WarningTimeDebounce WarningTimeDebounce = new(TimeSpan.FromSeconds(1));
+
+        // TODO: remove 'IEventCapture' in  further iteration
+        private readonly IEventCapture? _eventCapture;
+        private readonly IAppDomain _appDomain;
+
+        private IHub? _hub;
+
+        public UnityApplicationLoggingIntegration(IAppDomain? appDomain = null, IEventCapture? eventCapture = null)
+        {
+            _appDomain = appDomain ?? UnityAppDomain.Instance;
+            _eventCapture = eventCapture;
+        }
+
+        public void Register(IHub hub, SentryOptions _)
+        {
+            _hub = hub;
+
+            _appDomain.LogMessageReceived += OnLogMessageReceived;
+            _appDomain.Quitting += OnQuitting;
+        }
+
+        // Internal for testability
+        internal void OnLogMessageReceived(string condition, string stackTrace, LogType type)
+        {
+            var debounced = type switch
+            {
+                LogType.Error or LogType.Exception or LogType.Assert => ErrorTimeDebounce.Debounced(),
+                LogType.Log => LogTimeDebounce.Debounced(),
+                LogType.Warning => WarningTimeDebounce.Debounced(),
+                _ => true
+            };
+            if (!debounced)
+            {
+                return;
+            }
+
+            // TODO: to check against 'MinBreadcrumbLevel'
+            if (type != LogType.Error && type != LogType.Exception && type != LogType.Assert)
+            {
+                // TODO: MinBreadcrumbLevel
+                // options.MinBreadcrumbLevel
+                _hub?.AddBreadcrumb(condition, level: ToBreadcrumbLevel(type));
+                return;
+            }
+
+            var sentryEvent = new SentryEvent(new UnityLogException(condition, stackTrace));
+            sentryEvent.SetTag("log.type", ToEventTagType(type));

```

Comment:
Is this supposed to be `event.level` instead?

Reply:
I don't know what you mean. Can you elaborate?

Reply:
I mean, instead of adding a tag, we can just use `sentryEvent.Level = `

Reply:
What do you expect for `LogType.Assert`?

![image](https://user-images.githubusercontent.com/2175097/117266423-ead5be00-ae5d-11eb-87de-246d3bbc37e0.png)